### PR TITLE
VAGOV-974 accessibility tests

### PR DESCRIPTION
**Pull request by AndreyAndreev1380**
refactor to use env variables, added to composer va:test:accessibility, tests.yml.

-----------------------------------------
Tests are running with `lando test` locally. There are currently 4 violation types failing that we are ignoring right now, e.g. like duplicate IDs. The app team will need to look at to resolve but this is a great start that we can merge once CI passes. ~ Elijah

### CODE REVIEW

File: nightwatch.json [hunk 1]
```diff
@@ -12,15 +12,15 @@
         "log_path" : "",
         "port" : 9515,
         "cli_args" : {
-            "webdriver.chrome.driver" : "/usr/local/bin/chromedriver",
+            "webdriver.chrome.driver" : "./node_modules/chromedriver/lib/chromedriver",
```
Comment (ElijahLynn) [inline nightwatch.json]: We should delete this file.

**Comment by ElijahLynn** [comment]
This failed for two reasons:

**1 is a known flicker:**

> Attempting to load Drupal content from API at http://pr610.ci.cms.va.gov
>       │ FetchError: invalid json response body at http://pr610.ci.cms.va.gov/graphql reason: Unexpected end of JSON input
>       │     at /var/aegir/projects/cms/pr610/docroot/vendor/va-gov/web/node_modules/node-fetch/lib/index.js:254:32
>       │     at process._tickCallback (internal/process/next_tick.js:68:7)
>       │ Failed to pipe Drupal content into Metalsmith!

**2 is because we need to add the VA CA to trusted certs for Node**
The existing WEB invocation doesn't have this issue so I have to dig in to see why. But I think this is easy to solve.

File: nightwatch.json (hunk 1 above)
Comment (AndreyAndreev1380) [reply nightwatch.json]: doen

**Comment by ElijahLynn** [comment]
> ######  Test Failed: `va/tests/accessibility`
> ```shell
> composer va:test:accessibility
> ```
> 
> ```shell
> audited 183 packages in 1.047s
> found 2 low severity vulnerabilities
> run `npm audit fix` to fix them, or `npm audit` for details
> 
> > axe_tests@1.0.0 test /var/aegir/projects/cms/pr610
> > node ./tests/accessibility/aXeAccessibilityCheck.js
> 
> !!!  NUMBER OF NEW VIOLATIONS on http://pr610.ci.cms.va.gov 0
> !!!  NUMBER OF NEW VIOLATIONS on http://pr610.ci.cms.va.gov/sections  = 0
> []
> !!!  NUMBER OF NEW VIOLATIONS on http://pr610.ci.cms.va.gov/node/add/page  = 1
> [ { id: 'duplicate-id',
>   impact: 'minor',
>   tags: [ 'cat.parsing', 'wcag2a', 'wcag411' ],
>   description: 'Ensures every id attribute value is unique',
>   help: 'id attribute value must be unique',
>   helpUrl:
>    'https://dequeuniversity.com/rules/axe/3.3/duplicate-id?application=webdriverjs',
>   nodes: [ [Object], [Object] ] } ]
> !!!  NUMBER OF NEW VIOLATIONS on http://pr610.ci.cms.va.gov/node/add/landing_page  = 0
> []
> !!!  NUMBER OF NEW VIOLATIONS on http://pr610.ci.cms.va.gov/node/add/documentation_page  = 0
> []
> !!!  NUMBER OF NEW VIOLATIONS on http://pr610.ci.cms.va.gov/node/add/event  = 0
> []
> !!!  NUMBER OF NEW VIOLATIONS on http://pr610.ci.cms.va.gov/node/add/health_care_local_facility  = 0
> []
> !!!  NUMBER OF NEW VIOLATIONS on http://pr610.ci.cms.va.gov/node/add/health_care_region_detail_page  = 1
> [ { id: 'duplicate-id',
>   impact: 'minor',
>   tags: [ 'cat.parsing', 'wcag2a', 'wcag411' ],
>   description: 'Ensures every id attribute value is unique',
>   help: 'id attribute value must be unique',
>   helpUrl:
>    'https://dequeuniversity.com/rules/axe/3.3/duplicate-id?application=webdriverjs',
>   nodes: [ [Object], [Object] ] } ]
> !!!  NUMBER OF NEW VIOLATIONS on http://pr610.ci.cms.va.gov/node/add/health_care_region_page  = 0
> []
> !!!  NUMBER OF NEW VIOLATIONS on http://pr610.ci.cms.va.gov/node/add/office  = 0
> []
> !!!  NUMBER OF NEW VIOLATIONS on http://pr610.ci.cms.va.gov/node/add/outreach_asset  = 0
> []
> !!!  NUMBER OF NEW VIOLATIONS on http://pr610.ci.cms.va.gov/node/add/person_profile  = 1
> [ { id: 'duplicate-id-aria',
>   impact: 'critical',
>   tags: [ 'cat.parsing', 'wcag2a', 'wcag411' ],
>   description:
>    'Ensures every id attribute value used in ARIA and in labels is unique',
>   help: 'IDs used in ARIA and labels must be unique',
>   helpUrl:
>    'https://dequeuniversity.com/rules/axe/3.3/duplicate-id-aria?application=webdriverjs',
>   nodes: [ [Object] ] } ]
> !!!  NUMBER OF NEW VIOLATIONS on http://pr610.ci.cms.va.gov/node/add/press_release  = 1
> [ { id: 'aria-allowed-attr',
>   impact: 'critical',
>   tags: [ 'cat.aria', 'wcag2a', 'wcag412' ],
>   description: 'Ensures ARIA attributes are allowed for an element\'s role',
>   help: 'Elements must only use allowed ARIA attributes',
>   helpUrl:
>    'https://dequeuniversity.com/rules/axe/3.3/aria-allowed-attr?application=webdriverjs',
>   nodes: [ [Object] ] } ]
> !!!  NUMBER OF NEW VIOLATIONS on http://pr610.ci.cms.va.gov/node/add/regional_health_care_service_des  = 0
> []
> !!!  NUMBER OF NEW VIOLATIONS on http://pr610.ci.cms.va.gov/node/add/news_story  = 0
> []
> !!!  NUMBER OF NEW VIOLATIONS on http://pr610.ci.cms.va.gov/node/add/support_service  = 0
> []
> !!!  VIOLATION TYPES FOUND: 4 PROCESS EXITED WITH CODE 1  !!!
> 
> > npm install --only=production
> > npm test
> Wed, 02 Oct 2019 05:41:57 GMT axe-webdriverjs deprecated Error must be handled as the first argument of axe.analyze. See: #83 at tests/accessibility/aXeAccessibilityCheck.js:44:14
> Wed, 02 Oct 2019 05:41:59 GMT axe-webdriverjs deprecated Error must be handled as the first argument of axe.analyze. See: #83 at tests/accessibility/aXeAccessibilityCheck.js:56:42
> Failed to inject axe-core into one of the iframes!
> Failed to inject axe-core into one of the iframes!
> npm ERR! Test failed.  See above for more details.
> Script npm test handling the va:test:accessibility event returned with error code 1
> ```
> 
> _AUTOMATED COMMENT FROM provision-ops/yaml-tests, running on devshop.cms.va.gov_

I just triggered tests manually on this PR at http://devshop.cms.va.gov/node/22774

Nice to see this actually passed the tests AND posted this failures comment since we have `ignore-failure` on for that test in yaml-tests. I was hoping it would still post this comment even though we are ignoring failures. 

Second, this actually passed because I set `NODE_EXTRA_CA_CERTS: /etc/pki/tls/certs/ca-bundle.crt` in the CI generated .env file, on next push that will be wiped. I am working on getting that into the generation script now.

**Pull request merged**